### PR TITLE
Add doc for using an external postgresql server

### PR DESCRIPTION
**Pull request by bdunne**
Moving from https://github.com/ManageIQ/manageiq-pods/tree/master/manageiq-operator#configuring-external-postgres

### CODE REVIEW

File: installing_on_kubernetes/_topics/external_postgres.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+        $ oc create secret generic postgresql-secrets \
+          --from-literal=dbname=vmdb_production \
+          --from-literal=hostname=YOUR_HOSTNAME \
+          --from-literal=port=5432 \
+          --from-literal=password=YOUR_PASSWORD_HERE \
+          --from-literal=username=YOUR_USERNAME_HERE \
+          --from-literal=sslmode=verify-full \ # optional
+          --from-file=rootcertificate=path/to/optional/certificate.pem # optional
+
+Note: If desired, the secret name is also customizable by setting `databaseSecret` in the CR.
```
Comment (dehawkins512) [inline installing_on_kubernetes/_topics/external_postgres.md:14]: Avoid "desired" and use "wanted or needed".
Note: If wanted, the secret name is also customizable by setting `databaseSecret` in the CR.

File: installing_on_kubernetes/_topics/external_postgres.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+## External postgresql
+
+Running with an external Postgres server is an option, if you want the default internal Postgres you can skip this step. Additionally, if you want to secure the connection, you need to include the optional parameters `sslmode=verify-full` and `rootcertificate` when creating the secret. To do this, manually create the secret substituting your values before creating the CR.
```
Comment (dehawkins512) [inline installing_on_kubernetes/_topics/external_postgres.md:3]: Running with an external Postgres server is an option, if you want the default internal Postgres you can skip this step. Additionally, if you want to secure the connection, you need to include the optional parameters `sslmode=verify-full` and `rootcertificate` when you create the secret. To do this, manually create the secret and substitute your values before you create the CR.

**Review by dehawkins512 — CHANGES_REQUESTED** [review]

File: installing_on_kubernetes/_topics/external_postgres.md (hunk 1 above)
Comment (bdunne) [reply installing_on_kubernetes/_topics/external_postgres.md]: Done

File: installing_on_kubernetes/_topics/external_postgres.md (hunk 2 above)
Comment (bdunne) [reply installing_on_kubernetes/_topics/external_postgres.md]: Done

**Review by Fryguy — APPROVED** [review]

**Pull request merged**